<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Plot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,3">
    <item>
     <layout class="QGridLayout" name="sidebar" rowstretch="0,0,0,0,0,0,0,0,0,0,0,1">
      <item row="1" column="0">
       <widget class="QLabel" name="xsymbol_label">
        <property name="text">
         <string>Symbol</string>
        </property>
        <property name="buddy">
         <cstring>xsymbol</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="yheader">
        <property name="text">
         <string>Vertical Axis</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="xunitlabel">
        <property name="text">
         <string>Unit</string>
        </property>
        <property name="buddy">
         <cstring>xunit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="xsymbol">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="yunit_label">
        <property name="text">
         <string>Unit</string>
        </property>
        <property name="buddy">
         <cstring>yunit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="yunit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="xunit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="ysymbol_label">
        <property name="text">
         <string>Symbol</string>
        </property>
        <property name="buddy">
         <cstring>ysymbol</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="yname">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="ysymbol">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="xheader">
        <property name="text">
         <string>Horizontal Axis</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="xname_label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>xname</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="xname"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="yname_label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>yname</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="fit_header">
        <property name="text">
         <string>Fit Results</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="intercept">
        <property name="text">
         <string/>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="slope">
        <property name="text">
         <string/>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="alabel">
        <property name="text">
         <string>Slope:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="blabel">
        <property name="text">
         <string>Intercept:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="data_table">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="plot_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>34</height>
    </rect>
   </property>
   <addaction name="load_action"/>
   <addaction name="save_action"/>
   <addaction name="export_action"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="load_action">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="export_action">
    <property name="text">
     <string>Save &amp;figure</string>
    </property>
    <property name="shortcut">
     <string notr="true">Ctrl+E</string>
    </property>
   </action>
  </widget>
 <tabstops>
  <tabstop>xsymbol</tabstop>
  <tabstop>xname</tabstop>
  <tabstop>xunit</tabstop>
  <tabstop>ysymbol</tabstop>
  <tabstop>yname</tabstop>
  <tabstop>yunit</tabstop>
  <tabstop>data_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
